src/components/common/CommonButton/CommonButton.jsx

```diff
@@ -41,14 +42,14 @@ const Button = styled(Link)`
 		}
 	}
 
-	@media screen and (max-width: 960px) {
+	@media screen and (max-width: ${TABLET}) {
 		font-size: 1rem;
-		width: 30%;
+		width: 180;
```

Comment:
```suggestion
		width: 180px;
```
The px unit is missing here.